mat64/qr.go

```diff
@@ -86,7 +86,7 @@ func (m *Dense) RFromQR(qr *QR) {
 
 // QFromQR extracts the m×m orthonormal matrix Q from a QR decomposition.
 func (m *Dense) QFromQR(qr *QR) {
-	r, c := qr.qr.Dims()
+	r, _ := qr.qr.Dims()
 	m.reuseAs(r, r)
```

Comment:
Ah yes, hidden in the fold. As you'll have noticed, Exp uses an initial I as well and it's probably worth sharing the reuseAsZeroed between them.